popup.js

```diff
@@ -0,0 +1,94 @@
+var github_link = document.getElementById("github_link");
```

Comment:
Please comment this properly for increasing readability for the developers in the future